modules/nw-multus-add-pod.adoc

```diff
@@ -29,7 +29,11 @@ The pod must be in the same namespace as the additional network.
 ifdef::sriov[]
 [NOTE]
 =====
-If a network attachment is managed by the SR-IOV Network Operator, the SR-IOV Network Resource Injector adds the `resource` field to the `Pod` object automatically.
+The SR-IOV Network Resource Injector adds the `resource` field to the first container in a pod automatically.
+
+If you are using an Intel network interface controller (NIC) in Data Plane Development Kit (DPDK) mode, only the first container in your pod is configured to access the NIC. Your SR-IOV additional network is configured for DPDK mode if the `deviceType` is set to `vfio-pci` in the `SriovNetworkNodePolicy` object.
+
+You can workaround this issue by either ensuring that the container that needs access to the NIC is the first container defined in the `Pod` object or by disabling the Network Resource Injector. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1990953[BZ#1990953].
```

Comment:
I believe "workaround" is a noun only. If you want to make it a verb, I'd say "You can work around"

Reply:
@bergerhoffer oops thanks!